tests/e2e/pageObjects/mainPage.js

```diff
@@ -0,0 +1,18 @@
+import {config} from "../wdio.conf";
+export class MainPage {
+
+    //Elements
+
+
+    //Methods
+    navigateTo() {
+        return browser.url('/main');
+    }
+
+    waitForPageisLoaded() {
+        browser.waitUntil(function () {
```

Comment:
Please use arrow functions in this case. 
Can be replaced with `browser.waitUntil( () => browser.getUrl() === (config.baseUrl + "/main"));`